Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -945,6 +945,14 @@ def homTensorHomMap : (M →ₗ[R] P) ⊗[R] (N →ₗ[R] Q) →ₗ[R] M ⊗[R]
 
 variable {R M N P Q}
 
+/--
+Given any linear maps `f : M → Hom(P, Q)` and `g : N → Hom(S, T)`, there is a linear map
+`M ⊗ N → Hom(P ⊗ S, Q ⊗ T)` given by `m ⊗ n ↦ f m ⊗ g n`
```

Comment:
I think an explanation in terms of binary functions might be useful too, to explain the relation with `map`; that `map₂ f g (m1 ⊗ n1) (m2 ⊗ n2) = f m1 m2 ⊗ g m1 m2`.

Reply:
Thanks! I kept both version